.github/workflows/e2e.yml

```diff
@@ -2,7 +2,9 @@
 name: end-to-end
 
 on:
-  workflow_dispatch:
+  workflow_dispatch: # Activate this workflow manually
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Let's add this branch as a trigger before merging to verify that the workflow is working properly. 👍